pkg/web/web.go

```diff
@@ -251,7 +252,7 @@ func New(ctx context.Context, opts ...Option) (*Server, error) {
 			}),
 		),
 	)
-	root.PathPrefix("/api/").Handler(apiRouter)
+	router.PathPrefix("/api/").Handler(apiRouter)
```

Comment:
This should stay on the root router, otherwise we risk breaking some Enterprise APIs.